.github/workflows/build.yml

```diff
@@ -50,7 +50,34 @@ jobs:
 
       - name: Expose JSON data
         run: cp build/data.json dist/data.json
+        
+      - name: Download old artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: build.yml
+          workflow_conclusion: completed
+          name: data.json
+          path: build/old.json
+
+      - name: Compare both artifacts
+        run: (diff build/data.json build/old.json &&  echo "NOT OK" | grep "NOT OK") && false # will exit the workflow with an error, stopping next build steps
+      
+      - name: Download combined artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: all.json
+          path: build/all.json
 
+      - name: Combine artifacts
+        run: jq -s . build/all.json build/data.json > build/all.json
+      
+      - name: Upload a Build Artifact
+        uses: actions/upload-artifact@v2.2.2
+        with:
+          # Artifact name
+          name: all.json
+          path: build/all.json
+      
```

Comment:
You also need to upload data.json here so that it can be referenced in subsequent builds to check if the emote has changed?